aten/src/ATen/native/cuda/BatchLinearAlgebra.cu

```diff
@@ -1893,6 +1951,145 @@ std::tuple<Tensor, Tensor> _triangular_solve_helper_cuda(const Tensor& self, con
   return std::tuple<Tensor, Tensor>(self_working_copy, A_working_copy);
 }
 
+// ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ orgqr ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+/*
+  The orgqr function allows reconstruction of an orthogonal (or unitary) matrix Q,
+  from a sequence of elementary reflectors, such as produced by the geqrf function.
+
+  Args:
+  * `self` - Tensor with the directions of the elementary reflectors below the diagonal,
+              it will be overwritten with the result
+  * `tau` - Tensor containing the magnitudes of the elementary reflectors
+  * `infos` - Tensor to store MAGMA's error codes (not used here)
```

Comment:
MAGMA or cuSOLVER?

Reply:
MAGMA here because this function (and this file) is calling MAGMA.